src/definitions/behaviors/api.js

```diff
@@ -633,14 +633,14 @@ $.api = $.fn.api = function(parameters) {
                 response = responder;
               }
               // simulating response
-              mockedXHR.resolveWith(context, [ response, textStatus, { responseText: response }]);
+              mockedXHR.resolveWith(context, [response, textStatus, { responseText: response }]);
```

Comment:
Which rule in eslintrc.js is causing this?
Although i like removing bracket-spacing it "conflicts" with the "object-curly-spacing" rule, where a space is inserted instead of being removed. This seem inconsistent to me.
I suggest using `"space-in-brackets": ["error", "never"]` instead to remove trailing spaces in object and array lists  equally

Reply:
The below rules are related.
* standard/array-bracket-even-spacing
* standard/computed-property-even-spacing
* standard/object-curly-even-spacing

According these rule, spacing around array/computed-property/object should be even.

👍 No spacing

```js
mockedXHR.resolveWith(context, [response, textStatus, { responseText: response }]); 
```

👍 Spacing 
```js
mockedXHR.resolveWith(context, [ response, textStatus, { responseText: response } ]); 
```

👎 Inconsist spacing

```js
mockedXHR.resolveWith(context, [ response, textStatus, { responseText: response }]); 
```

Reply:
ok then :)